test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -1,650 +1,85 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
On a clean build, I'm still seeing 11 test failures consistently in the end to end tests.

Reply:
Can you share your failed tests list? They all pass in CI